### ISSUE TITLE
Unable to import data from browser

### ISSUE THREAD

**Issue by ghost** [MrAlex94/Waterfox#2442]
### What happened?

I am unable to import data from another browser due to this bug, see image.

### Reproducible?

- [ ] I have tested this on Firefox.

### Version

G3.𝑋 (Current)

### What platform are you seeing the problem on?

Windows

**Comment by ghost** [comment]
https://1drv.ms/u/s!AploExad608_g9kuRx4XcQOT-D4H1Q link to image

**Comment by Chwoka** [comment]
I'm using G4.0.3 on Windows 10 and I'm trying to import browser history from a file. (Unlike the fellow above me, my Import Wizard actually does give me options to click on.) Vivaldi's original History file has no written file extension, but secretly it is .sqlite (.db3 also works) so I can "fix" that just by copying and renaming the file. I've also converted it with a third-party tool (ChromeHistoryView) to .xml, .html, .htm, .csv, and .txt, which I didn't think would work if .sqlite didn't work, but I figured was worth a shot.

However, the file navigator opened by the Waterfox Import Wizard simply doesn't show any files of any kind at _all_ for me to select, only folders, and when I select the folder, it says it's "unable to identify any data to import" regardless of what format the data is in. Even though Chrome is a radio-button default option so the capability to read and convert its data must be there, the Import Wizard throws the exact same error if I manually navigate its file explorer to Google Chrome's User Data/Default/, which naturally is structured identically to Vivaldi's, so this isn't an issue of using a fringe browser. Navigating to my Firefox profile manually worked fine, at least. Perhaps the folder explorer is only recognizing Firefox-style data even though the Import Wizard purports to read Chrome data?

Another simple workaround that occurred to me was to put all my Vivaldi data in the filepath usually used for Google Chrome to trick the Import Wizard into looking for the right thing in the right place. It would let me click through each part of the dialog, but each step was just telling me to pick a profile or telling me it was going to import the following followed by big empty space, which didn't instill much faith that this was going to work, and indeed if I go all the way through the dialog it just seems to hitch. For what it's worth, this trick works on LibreWolf, though it still doesn't catch the browser history even though it says it will, and it seems to only pretend to work in the newest Windows version of Waterfox Classic, 2021.10, creating like an empty "From Google Chrome" folder in the bookmarks, though it's hard to tell since Library view seems to not display literally anything right now. I changed Chrome back to its unaltered state and the same thing happened. So does importing from Chrome data just flat out not work? Do I just gotta wait? Is this a Firefox thing?

**Comment by ghost** [comment]
@Chwoka I had the same issue, what I did was import browser stuff to Firefox then sync and use that same account in Waterfox. I also have the error you have too.

**Comment by RiversideGoldCorvette** [comment]
I just started using Waterfox, G4.0.6, and had the same issues with importing from Firefox. I was going to uninstall but did a refresh instead and all my bookmarks showed up. However, only about 70% of them work.  I've exported from Firefox to a .html several times and imported the file with bookmark manager getting the same results.  Since that didn't get all the bookmarks back, I opened a new tab, went to the desired website and tried to bookmark it - hit the star icon, CTR+D, drag&drop - but it refused to create a bookmark.  Using the bookmark manager, I manually created a bookmark entry and typed in the address.  It accepted the name but refused the URL  If I misspell the URL, it accepted it.  Very Strange!   This is the URL
https://www.5speeds.com/muncie.htm   Bookmark Manager will not save this URL.  If it is misspelled in any way, then it will save it.

**Comment by jahnson** [comment]
I have the same problem.  I exported bookmarks from WF classic to a .json file.  
In WF4.0.6, I tried to import bookmarks, but when I select "Import from file" it only allows me to select a directory, and then it says it is "Unable to identify any data to import from: Waterfox3".  

WF4.0.6 also allows me to select another browser to import from.  However, when I select WF classic, Next, then "Select which items to import:" is blank.
